### PR TITLE
Rename 'Google Cloud Debugger' to 'Stackdriver Debug'

### PR DESCRIPTION
**Pull request by patflynn**
addresses some of #936

### CODE REVIEW

File: google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties [hunk 1]
```diff
@@ -192,7 +192,7 @@ clouddebug.warninglabel=Warning\:
 clouddebug.attachingtext=Attaching...
 clouddebug.stopandcontinue=Stop IDE Debugging.
 clouddebug.exitdebug=Exit debug mode and continue the debug session later.
-clouddebug.message.title=Cloud Debugger
+clouddebug.message.title=Stackdriver Debugger
```
Comment (joaoandremartins) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: Should we remove this one and update references to "clouddebug.stackdriver.debugger" above?

**Comment by joaoandremartins** [comment]
LGTM besides the comment

**Comment by patflynn** [comment]
@joaoandremartins you need to specifically approve the PR.

File: google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties (hunk 1 above)
Comment (patflynn) [inline google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties]: Yeah probably. I think the whole labelling strategy is a bit of a mess right now. I'm not going to fix this here though.

**Pull request merged**